build-system/tasks/build.js

```diff
@@ -30,23 +29,30 @@ const {compileCss} = require('./css');
 const {createCtrlcHandler, exitCtrlcHandler} = require('../ctrlcHandler');
 const {isTravisBuild} = require('../travis');
 const {parseExtensionFlags} = require('./extension-helpers');
+const {serve} = require('./serve');
+const {updatePackages} = require('./update-packages');
 
-const maybeUpdatePackages = isTravisBuild() ? [] : ['update-packages'];
 
 /**
  * Enables watching for file changes in css, extensions.
  * @return {!Promise}
  */
-function watch() {
-  const handlerProcess = createCtrlcHandler('watch');
-  return performBuild(true).then(() => exitCtrlcHandler(handlerProcess));
+async function watch() {
+  if (!isTravisBuild()) {
```

Comment:
Since this is repeated a few times, would it make sense to abstract into a method either locally or in the `update-packages` module?

Reply:
Good idea! Done.